docs/.vscode/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+    "grammarly.selectors": [
```

Comment:
what will this do? @saahilthukral

Reply:
This is just a plugin for grammarly vscode. I pushed this file by accident